src/components/RecipeModal/RecipeModal.jsx

```diff
@@ -19,32 +36,44 @@ export default function RecipeModal(props) {
           <Container>
             {/* first row - recipe overview */}
             <Row className="modal-overview">
-                <Col>
-                Total Time: {recipeInfo.readyInMinutes} mins
-                </Col>
-                <Col>
-                Servings: {recipeInfo.servings}
-                </Col>
+              {validMins 
```

Comment:
Since we're not displaying anything if validMins isn't true we can do 
{validMins && (<Col> Total Time: {recipeInfo.readyInMinutes} mins </Col>)}